calico_node/filesystem/sbin/check_readiness

```diff
@@ -0,0 +1,46 @@
+#!/bin/sh
+
+# Check felix readiness
+if [ ! -z "$FELIX_CHECKREADINESS" ]; then
```

Comment:
>We probably shouldn't define new FELIX_ variables here
Absoultely agreed. I think I'm going crazy here - I could have sworn this was the variable I found in our manifests but looks like you're right - its HEALTHENABLED.

As for your other two points, if they want to disable readiness checks, they can just remove them from the manifest. Since there's no "service" running for these, its simpler than the Felix ones in that regard.